README.md

```diff
@@ -1,3 +1,16 @@
 # URL shortener
 
 Greetings reviewers!
+
+## Installation and Usage
+
+- clone the repo and `cd` into the folder
+- to start the server:
+  - `cd` into the backend folder
+  - run `pipenv install` and `pipenv install --dev`
+  - run `pipenv shell`
+  - run `pipenv run dev`
```

Comment:
maybe this is an issue on my side but I followed all the steps and i got this error "/bin/bash: flask: command not found", either way try to test this out on your side to see if this is a real issue or just a me issue)

Dafé tried and his works so it turns out this is just a me issue.

Reply:
yeah I've never seen it before, maybe because it was set up on mac?

Reply:
Does it work if you switch the order of shell and install, I'm still not sure the way around it is supposed to be